source-mysql/replication.go

```diff
@@ -468,10 +468,10 @@ func (rs *mysqlReplicationStream) handleQuery(ctx context.Context, schema, query
 	//     don't impact our capture or because we get the relevant information
 	//     by some other means.
 	if ignoreQueriesRe.MatchString(query) {
-		logrus.WithField("query", query).Trace("ignoring query event")
+		logrus.WithField("query", query).Info("ignoring query event")
 		return nil
 	}
-	logrus.WithField("query", query).Debug("handling query event")
+	logrus.WithField("query", query).Info("handling query event")
```

Comment:
It shouldn't, at least in the common case. Because we require the user to set `binlog_format=ROW` we should only ever see DML operations as row-change events, and in fact seeing an insert or delete query event on a captured table, or (because of technical reasons and laziness) any update query events at all, is a fatal error.

It is theoretically possible for a user to have a really screwy setup where some DML operations on non-captured tables are written to the binlog verbatim, but in practice I'm not too worried about that and even if such a setup exists I'm willing to bet such query events would be very rare.

TL;DR: Barring some rare edge cases, as a general rule only DDL operations are written to the binlog as query events so we'll only be logging DDL queries as a result of this change.